fastlane/docs/Actions.md

```diff
@@ -2487,4 +2487,20 @@ To run [danger](https://github.com/danger/danger) add this to your `Fastfile`
 lane :danger do
   danger
 end
+=======
```

Comment:
Sorry!. It maybe happened when handling git rebase conflict, and I just fixed it.